### PR TITLE
Added text on creating an image pull secret for the tutorial

### PR DESCRIPTION
**Pull request by matsbror**
This pull requests outlines that you now need an image pull secret in Kubernetes and how you create it for the Azure container registry.

### CODE REVIEW

File: docs/intro/tutorial02.md [hunk 1]
```diff
@@ -131,6 +131,61 @@ $ wasm-to-oci push demo.wasm mycontainerregistry007.azurecr.io/krustlet-tutorial
 1](tutorial01.md) of this tutorial. If you are publishing the Rust example, use
 `target/wasm32-wasi/debug/demo.wasm` instead.
 
+## Create a container registry pull secret 
+
+Unless your container registry is enabled with anonymous access, you need to authenticate krustlet to pull images from it. At the moment, there is no flag in the Azure portal to make a registry public, but you can [create a support ticket](https://docs.microsoft.com/en-us/azure/container-registry/container-registry-faq#how-do-i-enable-anonymous-pull-access) to have it enabled manually. 
+
+Without public access to the container registry, you need to create a _Kubernetes pull secret_. The steps below for Azure are [extracted from the Azure documentation](https://docs.microsoft.com/en-us/azure/container-registry/container-registry-auth-kubernetes), and repeated herefor convenience.
```
Comment (thomastaylor312) [inline docs/intro/tutorial02.md:138]: s/herefor/here for/

**Review by thomastaylor312 — CHANGES_REQUESTED** [review]
Thank you so much for adding this! Just one small spelling thing and if you could please line wrap at 80 characters, that would be great!

File: docs/intro/tutorial02.md (hunk 1 above)
Comment (matsbror) [reply docs/intro/tutorial02.md:138]: Will do

Comment (matsbror) [reply docs/intro/tutorial02.md]: Done, some http-links still remain long.

Comment (thomastaylor312) [reply docs/intro/tutorial02.md]: HTTP links should be just fine!

**Review by thomastaylor312 — APPROVED** [review]
Thanks again!

**Pull request merged**